src/styles/main.scss

```diff
@@ -1,3 +1,170 @@
-body {
+// #region page
+.page {
+  font-family: Manrope, sans-serif;
+  color: #1B2129;
```

Comment:
It's better not to use single colour name, but to use constant which represents the colour.